config/servicemappings/alloydb.yaml

```diff
@@ -105,7 +105,6 @@ spec:
         - tfField: network
           description: |-
             Only `external` field is supported to configure the reference.
-            
```

Comment:
I think we added an empty line in between the "Only `external` field is supported to configure the reference." and the actual description for all the cases:
https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/config/servicemappings/cloudbuild.yaml
https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/config/servicemappings/alloydb.yaml

If you find it a suboptimal UI, then we probably want to change all of them?

Reply:
Thanks! I added an empty line.

It doesn't seem to make a difference in the rendered page though. 

https://cloud.google.com/config-connector/docs/reference/resource-docs/cloudbuild/cloudbuildtrigger

Reply:
Thank you for verifying it, Shuxian! Yes, it seems that the rendered doc is just plain text and doesn't keep any of the formats. Though it does make it clear in the CRD yamls. Let's just keep it for consistency.